tools/model_tools/README.md

```diff
@@ -27,15 +27,23 @@ future releases.
 > <br>
 > Proceed to an [easy installation from Docker](@ref workbench_docs_Workbench_DG_Run_Locally) to get started.
 
-## Prerequisites
+## Installation
 
-1. Install Python (version 3.6 or higher)
-2. Install the tools' dependencies with the following command:
+Model Downloader and other automation tools can be installed as a part of OpenVINO&trade; toolkit or from source.
+Installation from source is as follows:
+
+1. Install Python (version 3.6 or higher), [setuptools](https://pypi.org/project/setuptools/):
+
+2. Install the tools with the following command:
 
 ```sh
-python3 -mpip install --user -r ./requirements.in
+python setup.py build
```

Comment:
Do we really need `python setup.py build`? `install` options covers this step.

Reply:
`install` option doesn't cover coping [open_model_zoo/models ](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models)and [open_model_zoo/data](https://github.com/openvinotoolkit/open_model_zoo/tree/master/data) folders to the package, since they are outside the project directory. These are needed for downloading and converting models.

Reply:
Consider to overload 'build_py' class instead of 'build':

```
26  import distutils.command.build    -> import distutils.command.build_py

57 class CustomBuild(distutils.command.build.build):    -> class CustomBuild(distutils.command.build_py.build_py):  

103   cmdclass={                         ->          cmdclass={
           'build': CustomBuild,                             'build_py': CustomBuild,
```

Reply:
Thanks, done